libs/@blockprotocol/graph/src/shared/stdlib/subgraph/edge/link-entity.ts

```diff
@@ -187,12 +184,7 @@ export const getIncomingLinksForEntity = <Temporal extends boolean>(
             );
 
             if (intersection === null) {
-              throw new Error(
-                `No entity revision was found which overlapped the given edge, subgraph was likely malformed.\n` +
-                  `EntityId: ${linkEntityId}\n` +
-                  `Search Interval: ${JSON.stringify(searchInterval)}\n` +
-                  `Edge Valid Interval: ${JSON.stringify(edgeInterval)}`,
-              );
+              continue;
             }
```

Comment:
@CiaranMn we only patched this for `getOutgoingLinksForEntity`, but don't see why we shouldn't do this for `getIncomingLinksForEntity`